README.md

```diff
@@ -50,3 +50,45 @@ To run the tests using Poetry and Pytest, stop the app running above and run the
 ```bash
 $ poetry run pytest
 ```
+
+## Running the application using docker
+Use following commands to run docker image todo-app
+
+## Running the application in single-stage docker file
+```bash
+docker build -t todo-app .
+
+Pass environment variables as a file, publish the application on port 5000 and listen on port 5000
+docker run --env-file .env -p 5000:5000 
+```
```

Comment:
You don't have a single-stage Dockerfile, so this section of the README should be deleted.